### PR TITLE
Tidy up build

### PR DESCRIPTION
**Pull request by lauraluiz**
- Remove Docker publishing (Sunrise Starter Project takes care of that)
- Release only modules that need releasing
- Publish Javadoc only on released modules

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -2,9 +2,9 @@ import java.text.SimpleDateFormat
 import java.util.Date
 
 import play.sbt.PlayImport
-import sbt.Keys._
+import sbt.Keys.{publishArtifact, _}
```
Comment (schleichardt) [inline build.sbt]: the old import was better

Comment (lauraluiz) [inline build.sbt]: It was Intellij... but don't review yet, I'm still working on it ;)

File: project/Release.scala [hunk 2]
```diff
@@ -6,44 +6,48 @@ import com.typesafe.sbt.pgp.PgpKeys
 object Release {
   private val pathToPgpPassphrase = System.getProperty("user.home") + "/.sbt/gpg/passphrase"
 
+  lazy val disablePublish: Def.Setting[Boolean] = publishArtifact := false
+
+  lazy val enableSignedRelease = Seq(
+    releasePublishArtifactsAction := PgpKeys.publishSigned.value,
+    PgpKeys.pgpPassphrase in Global := {
```
Comment (lauraluiz) [inline project/Release.scala]: FYI: Like this, it wouldn't work if I put `in ThisBuild`

**Comment by lauraluiz** [comment]
@schleichardt finally, please review

File: build.sbt [hunk 3]
```diff
@@ -1,257 +1,77 @@
...
+val childProjects: List[sbt.ProjectReference] =
+  List(common, `product-catalog`, `shopping-cart`, `my-account`, `move-to-sdk`, `sbt-tasks`)
 
 lazy val `commercetools-sunrise` = (project in file("."))
-  .enablePlugins(PlayJava, DockerPlugin).configs(IntegrationTest, PlayTest)
-  .settings(commonSettings ++ commonTestSettings ++ sunriseThemeSettings ++ dockerSettings: _*)
-  .dependsOn(commonWithTests, `product-catalog`, `shopping-cart`, `my-account`)
-  .aggregate(common, `product-catalog`, `shopping-cart`, `my-account`, `sbt-tasks`, `move-to-sdk`)
+  .enablePlugins(PlayJava)
+  .settings(javadocSettings ++ Release.disablePublish: _*)
```
Comment (schleichardt) [inline build.sbt]: no need to change it , but splitting it into different settings calls is more consistent

```scala
.settings(javadocSettings: _*)
.settings(Release.disablePublish: _*)
```

File: project/Dependencies.scala [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import play.sbt.PlayImport.{cache, filters}
+import sbt.Keys._
+import sbt._
+
+object Dependencies {
+  private lazy val sunriseThemeVersion = "0.61.1"
```
Comment (schleichardt) [inline project/Dependencies.scala]: no need for `lazy` in String values

Comment (lauraluiz) [inline project/Dependencies.scala]: True that

**Pull request merged**